src/ActionList/Item.tsx

```diff
@@ -0,0 +1,88 @@
+import type {IconProps} from '@primer/octicons-react'
+import React from 'react'
+import styled from 'styled-components'
+import {get} from '../constants'
+
+interface ItemPropsBase extends React.ComponentPropsWithoutRef<'div'> {
+  text: string
+  description?: string
+  descriptionVariant?: 'inline' | 'block'
+  leadingVisual?: React.FunctionComponent<IconProps>
+  leadingVisualSize?: 16 | 20
+  size?: 'small' | 'medium' | 'large'
+  variant?: 'default' | 'singleSelection' | 'multiSelection' | 'danger' | 'static'
+}
+interface ItemPropsWithRenderItem extends Partial<ItemPropsBase> {
+  renderItem: (props: ItemProps) => JSX.Element
+}
+export type ItemProps = ItemPropsBase | ItemPropsWithRenderItem
+
+const StyledItem = styled.div<{variant: ItemProps['variant']}>`
+    position: relative;
```

Comment:
It's fine if required, but why does this need to be `relative`?

Reply:
Removed in https://github.com/primer/components/pull/1078/commits/1164334e8e23510560a6ab748f8c7f4d50ba14e0!